MWSE/LuaObject.h

```diff
@@ -576,6 +577,81 @@ namespace mwse::lua {
 		}
 	};
 
+	template<>
+	class ObjectCreator<TES3::Book> : public ObjectCreatorBase {
+	public:
+		TES3::BaseObject* create(sol::table params, bool getIfExists) const override {
```

Comment:
`getIfExists` does not get checked.

Reply:
Looks like `enchantment` has the same issue that I copied over...

Reply:
That is correct. Ill fix that for the enchantment, thanks.